### PR TITLE
TEMP PR

### PR DESCRIPTION
**Pull request by edalgrin**
TEMP PR

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_tag_selector.scss [hunk 1]
```diff
@@ -1,83 +0,0 @@
...
-	.textboxlist-input-container {
-		input {
-			margin: 0;
-		}
-
-		.textboxlist-content .field-input-text {
-			padding: 0;
-		}
-	}
-}
```
Comment (edalgrin) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_tag_selector.scss:35]: This is still used in several places

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss [hunk 2]
```diff
@@ -1,7 +1,3 @@
-.undomanager .lfr-undo-queue {
-	margin: 10px auto;
-}
-
```
Comment (edalgrin) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss:4]: Seems like the removed classes are still in use in https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/undo_manager.js

**Review by edalgrin — COMMENTED** [review]
@marcoscv-work Please take a look at the commented lines 👇 and let me know what you think!

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss (hunk 2 above)
Comment (edalgrin) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss:4]: They might be replaceable with utilities

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_generic_portlet.scss [hunk 3]
```diff
@@ -1,22 +0,0 @@
...
-	.controls-hidden {
-		.lfr-meta-actions,
-		.lfr-configurator-visibility {
-			display: none;
-		}
-	}
-
-	.page-maximized .portlet-topper-toolbar {
-		display: block;
-	}
```
[the remark this answers is not in the document]
Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_generic_portlet.scss:62]: Some customers are using this class to recognize if the portlet is on maximized state, for classes like this probably the best is keeping the classname we know customers are using, add the `d-block` and remove the CSS

[the remark this answers is not in the document]
Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_generic_portlet.scss:62]: nah, in this case is a subclass, go to remove it all

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_misc.scss [hunk 4]
```diff
@@ -10,6 +10,10 @@
 	}
 }
 
+.lfr-checkbox-preselected {
+	opacity: 0.5;
+}
+
```
[the remark this answers is not in the document]
Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_misc.scss:16]: it is really notice in the interface, for me it is totally unneeded, test will fail for sure in this case, not a big deal for me!

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_tag_selector.scss (hunk 1 above)
Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_tag_selector.scss:35]: after my review I think we can remove them, yeah!

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss (hunk 2 above)
Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_undo_manager.scss:4]: yeah!

**Comment by edalgrin** [comment]
https://github.com/liferay-frontend/liferay-portal/pull/1700

**Pull request closed without merging**